package.json

```diff
@@ -46,6 +47,7 @@
     "gulp-rename": "^1.2.0",
     "gulp-uglify": "^1.1.0",
     "istanbul": "^0.3.5",
+    "jayson": "^1.2.2",
```

Comment:
BTW, I didn't expose http server in public api, so it can't be used by user directly, only as example